app/models/media_attachment.rb

```diff
@@ -25,10 +25,10 @@ class MediaAttachment < ApplicationRecord
 
   enum type: [:image, :gifv, :video, :unknown]
 
-  IMAGE_FILE_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.gif'].freeze
+  IMAGE_FILE_EXTENSIONS = ['.jpg', '.jpeg', '.png', '.gif', '.webp'].freeze
```

Comment:
How about this: Keep the support but remove the changes to the tooltip. Then it's future-proof without encouraging users to upload images that still aren't supported by Safari/iOS

Reply:
@Gargron Thank you for the review! I've removed the changes at 993e3e059dde767a1fc3795434823e47a2fba087. Is this alright with you?